providers/vault/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/knadh/koanf/providers/vault
+module github.com/wangkang/koanf/providers/vault
```

Comment:
Hi. Why is the path changed? It should point to the the original repo where the fix is being added.

Reply:
Sorry, it is an unexpected commit